themis/languages/java/installation-desktop.md

```diff
@@ -177,5 +177,5 @@ so you do not have to install Gradle manually.
 The resulting JAR file `java-themis-X.Y.Z.jar` will be placed into `src/wrappers/themis/java/build/libs`.
 You then need to add it to your project.
 
-See [JavaThemis examples](https://github.com/cossacklabs/themis-java-examples)
+See [JavaThemis examples](../../java/examples/)
 for illustration of how to integrate JavaThemis into your project.
```

Comment:
This URL leads explicitly into Java section because the file is symlinked into Kotlin section as well. But since this is installation for desktop systems, we want to point users to an example for desktop systems, not Android.